ivy/functional/backends/numpy/extensions.py

```diff
@@ -285,3 +285,25 @@ def moveaxis(
 
 
 moveaxis.support_native_out = False
+
+
+def heaviside(
+    x1: np.ndarray,
+    x2: np.ndarray,
+    /,
+    *,
+    out: Optional[np.ndarray] = None,
+) -> np.ndarray:
+    return np.heaviside(
+        x1,
+        x2,
+        out=out,
+        where=True,
```

Comment:
We don't need to call on these args as they are set by default for numpy.
```python
        where=True,
        casting='same_kind',
        order='K',
        dtype=None,
        subok=True,
```

UNLESS there was some instructions updates that I missed or something then please feel free to notify me.

Reply:
There area already remved so no need to concern with. Thanks.